aspnetcore/grpc/authn-and-authz.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Authentication and authorization in gRPC for ASP.NET Core
+author: jamesnk
+description: Learn how to use authentication and authorization in gRPC for ASP.NET Core.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: jamesnk
+ms.date: 06/07/2019
+uid: grpc/authn-and-authz
+---
+
+# Authentication and authorization in gRPC for ASP.NET Core
+
+By [James Newton-King](https://twitter.com/jamesnk)
+
+[View or download sample code](https://github.com/aspnet/AspNetCore.Docs/tree/master/aspnetcore/grpc/authn-and-authz/sample/) [(how to download)](xref:index#how-to-download-a-sample)
+
+## Authenticate users calling a gRPC service
+
+gRPC can be used with [ASP.NET Core authentication](xref:security/authentication/identity) to associate a user with each call. 
```

Comment:
Even though this document only calls out JWT and Client cert, I'd like to see it mentioned that all ASP.NET Core supported authentication mechanisms like OAuth, OpenID, and Negotiate should all work